src/auction.js

```diff
@@ -150,16 +150,28 @@ export function newAuction({adUnits, adUnitCodes, callback, cbTimeout, labels})
     }
   }
 
+  function hasVideoBids(bids) {
+    const bidsArray = bids ? (bids[0] ? bids : [bids]) : [];
+    const videoBid = bidsArray.some(bid => !!(utils.deepAccess(bid.mediaTypes, ['video'])));
+    const cacheEnabled = !!(config.getConfig('cache.url'));
+
+    // video bids with cache enabled need to be cached first before they are considered done
```

Comment:
I don't think this is quite right.  It's possible for video bids to skip the cache by returning the `bid.videoCacheKey` and `bid.vastUrl` themselves, even if the cache is enabled.  So this logic needs to mirror the logic in `tryAddVideoBid` when it decides whether to handle done or not.  Ideally that logic should probably be pulled out of `tryAddVideoBid` so it could be used there and here to be sure we don't have discrepancies, IMO.